ament_flake8/ament_flake8/main.py

```diff
@@ -173,7 +174,36 @@ def get_flake8_style_guide(argv):
     return StyleGuide(application)
 
 
+def parse_config_file(config_file):
+    from flake8.options import config, manager, aggregator
+
+    opts_manager = manager.OptionManager(prog='flake8', version='3.0.0')
```

Comment:
Why 3.x? In Ubuntu 22.04 the [default flake8 version](https://packages.ubuntu.com/jammy/flake8) is 4.x.